apps/predbat/predbat.py

```diff
@@ -8650,9 +8650,10 @@ def fetch_sensor_data(self):
                         start = event.get("start", None)
                         end = event.get("end", None)
                         saving_rate = event.get("octopoints_per_kwh", saving_rate * octopoints_per_penny) / octopoints_per_penny  # Octopoints per pence
-                        if code:
+                        if code:  # Join the new Octopus saving event and send an alert
                             self.log("Joining Octopus saving event code {} start {} end {} price per kWh {}".format(code, start, end, saving_rate))
                             self.call_service("octopus_energy/join_octoplus_saving_session_event", event_code=code, entity_id=entity_id)
+                            self.base.call_notify("Predbat: Joined Octopus saving event: start {}, end {}, price per kWh {}".format(start, end, saving_rate))
```

Comment:
Do you think this should be enabled with 'self.set_status_notify' as per the status changes?

Reply:
I did wonder about that.  I concluded not, so the alert would always be sent, on the basis that:
1. Once comfortable with predbat users may be running it in the background and not receiving any notifications as a result
2. This was a significant "activity" that changes the way Predbat would behave, and users may not otherwise have been aware of the saving session and would want to be notified so they can change any other consumption behaviour (and potentially turn predbat off if they don't want the force discharge) - in effect it was a 'high priority' alert

Reply:
> I did wonder about that. I concluded not, so the alert would always be sent, on the basis that:
> 
> 1. Once comfortable with predbat users may be running it in the background and not receiving any notifications as a result
> 2. This was a significant "activity" that changes the way Predbat would behave, and users may not otherwise have been aware of the saving session and would want to be notified so they can change any other consumption behaviour (and potentially turn predbat off if they don't want the force discharge) - in effect it was a 'high priority' alert

Okay for me